instrumentation/opentelemetry-instrumentation-aiopg/src/opentelemetry/instrumentation/aiopg/aiopg_integration.py

```diff
@@ -123,7 +127,7 @@ async def traced_execution(
 
 
 def get_traced_cursor_proxy(cursor, db_api_integration, *args, **kwargs):
-    _traced_cursor = AsyncTracedCursor(db_api_integration)
+    _traced_cursor = AsyncCursorTracer(db_api_integration)
 
     # pylint: disable=abstract-method
     class AsyncTracedCursorProxy(AsyncProxyObject):
```

Comment:
I think this one should also be renamed to match above, `AsyncCursorTracerProxy`